powersimdata/network/usa_tamu/tests/test_model.py

```diff
@@ -0,0 +1,65 @@
+import pytest
+
+from powersimdata.network.usa_tamu.model import TAMU, check_and_format_interconnect
+
+
+def test_interconnect_type():
+    interconnect = {"Western", "Texas"}
+    with pytest.raises(TypeError):
+        check_and_format_interconnect(interconnect)
+
+
+def test_interconnect_value():
+    interconnect = ["Canada"]
+    with pytest.raises(ValueError):
+        check_and_format_interconnect(interconnect)
+
+
+def test_interconnect_duplicate_value():
+    interconnect = ["Western", "Western", "Texas"]
+    with pytest.raises(
+        ValueError, match="List of interconnects contains duplicate values"
+    ):
+        check_and_format_interconnect(interconnect)
+
+
+def test_interconnect_usa_is_unique():
+    interconnect = ["Western", "USA"]
+    with pytest.raises(ValueError, match="USA cannot be paired"):
+        check_and_format_interconnect(interconnect)
+
+
+def test_interconnect():
+    arg = ("Western", ["Eastern", "Western"])
+    expected = (["Western"], ["Eastern", "Texas"])
+    for a, e in zip(arg, expected):
+        check_and_format_interconnect(a) == e
```

Comment:
There should be an `assert` here, which would catch the fact that the `expected` above should be (["Western"], ["Eastern", "Western"]).

Reply:
Good catch

Reply:
Done